src/main/java/dao/ChessGamesDao.java

```diff
@@ -0,0 +1,36 @@
+package dao;
+
+import db.DBConnection;
+import dto.ChessGameVo;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+public class ChessGamesDao {
+    public ArrayList<ChessGameVo> findAll() throws SQLException {
+        String query = "SELECT * FROM game;";
+
+        try(Connection connection = DBConnection.getConnection();
+            PreparedStatement pstmt = connection.prepareStatement(query)) {
+            ResultSet rs = pstmt.executeQuery();
+            ArrayList<ChessGameVo> chessGameVos = getChessGameDtos(rs);
+            return chessGameVos;
+        }
```

Comment:
```suggestion
        try(Connection connection = DBConnection.getConnection();
            PreparedStatement pstmt = connection.prepareStatement(query);
            ResultSet rs = pstmt.executeQuery()) {
            ArrayList<ChessGameVo> chessGameVos = getChessGameDtos(rs);
            return chessGameVos;
        }
```
`ResultSet`의 자원을 회수하는건 어떨까요?